driver/src/main.rs

```diff
@@ -55,8 +57,13 @@ fn main() {
     let optimization_model = OptimizationModel::from(optimization_model_string.as_str());
 
     let web3 = web3_provider(&ethereum_node_url).unwrap();
-    let contract = BatchExchange::new(&web3, network_id).unwrap();
-    info!("Using contract at {}", contract.address());
+    let gas_station = GasPriceEstimatingImpl::new(
+        Duration::from_secs(10),
```

Comment:
We can read this from the environment or even use the same timeout as transport?

Reply:
That's true. I want to do this after we merge the environment and command line unification PR.